modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.scss

```diff
@@ -0,0 +1,3 @@
+.actions-dropdown-renderer-loading-indicator {
+	display: initial;
+}
```

Comment:
If we remove the buttons around the loading indicators, this style is not needed

Reply:
Hello everyone, I'm not in sync now because I'm in LA. I'll be back on the 4th of March. I tried to answer twice yesterday, but Github was down. 

So, loading indicators can be placed within buttons on certain occasions like text and icon + text buttons, even icon buttons. However, it makes sense not to use a button to place the loading indicator because there has been no button before. So the loading indicator is good enough to represent something is being processed. 

Anyway, It would be great to know all the steps to reproduce that behavior. Maybe you can create a request to Lexicon or share the Figma file to understand the case thoroughly.

Reply:
Hey @drakonux, can you please take a look, this has been stuck for a while. You can see steps to reproduce in [the JIRA ticket](https://issues.liferay.com/browse/LPS-138620).

Reply:
I tried to reproduce the issue on my local computer but I wasn't able to reproduce it. Could someone upload a video/gif of the issue?

Reply:
Hi @drakonux, I will send you a video on Slack